build-parent/pom.xml

```diff
@@ -75,7 +75,7 @@
         <microprofile-graphql-api.version>1.1.0</microprofile-graphql-api.version>
 
         <!-- Antlr is used by the PanacheQL parser-->
-        <antlr.version>4.8</antlr.version>
+        <antlr.version>4.9.2</antlr.version>
```

Comment:
Looks like a duplicate? If it's in the BOM, it could probably be removed from the build-parent.

Reply:
It's used for `antlr4-maven-plugin`, so the BOM is not of help here.

Reply:
It could be moved to the root parent then.

Reply:
The version property? So that there is only one property, also for the BOM? Or just for the plugin?

Reply:
Yes, to have it in one place. WDYT?

Reply:
Well, this _should_ work, but then again it would be a paradigm shift because currently, all BOM-relevant properties are set only in the BOM.

Reply:
Yeah, we can discuss changing how we do things but that's for another day. For now, properties are either in the BOM or in the build-parent.

Maybe having all of them in the parent would indeed be an improvement but let's discuss this separately.